index.js

```diff
@@ -61,4 +63,18 @@ const giveRatio = () => {
 foregroundColor.oninput = giveRatio;
 backgroundColor.oninput = giveRatio;
 
+const showColor = () => {
+  let firstColor = foregroundColor.value;
+  let secondColor = backgroundColor.value;
+  if (firstColor.length === 7) {
+    leftSwatch.style.backgroundColor = firstColor;
+  }
+  if (secondColor.length === 7) {
+    rightSwatch.style.backgroundColor = secondColor;
+  }
+};
+
+foregroundColor.oninput = showColor;
+backgroundColor.oninput = showColor;
```

Comment:
When I pulled this down and tested it, the contrast ratio was no longer showing.
Also, we have a lot of repetition between the `showColor` and `giveRatio` functions.
It might be better to create one function, and build off of it/ refactor it from there.

Something like this could work
```
const displayResult = () => {
  let firstColor = foregroundColor.value;
  let secondColor = backgroundColor.value;

  if (firstColor.length === 7) {
    leftSwatch.style.backgroundColor = firstColor;
  }

  if (firstColor.length === 7 && secondColor.length === 7) {
    rightSwatch.style.backgroundColor = secondColor;
    ratioResult.innerHTML = twoHexesRatio(firstColor, secondColor);
  }
};

foregroundColor.oninput = displayResult;
backgroundColor.oninput = displayResult;
```
<img width="566" alt="Screen Shot 2022-08-11 at 7 01 30 AM" src="https://user-images.githubusercontent.com/67210629/184151262-f9a6623d-92b2-4ddb-bfb8-9cec6fdc3d4f.png">